crates/holochain_types/src/app/app_manifest/app_manifest_v1.rs

```diff
@@ -152,18 +152,10 @@ pub enum CellProvisioning {
     /// Always create a new Cell when installing this App
     Create { deferred: bool },
 
-    /*
-        TODO: implement
+    /// Require that a Cell is already installed which has a DNA that's compatible with the
+    /// `installed_hash` specified in the manifest.
+    UseExisting { deferred: bool },
```

Comment:
What does `deferred` mean in the `UseExisting` case?

This field should probably have a doc comment in the `Create` above too?

Reply:
This is updated in the followup PR: https://github.com/holochain/holochain/blob/293d6e775b3f02285b831626c9911802207a8d85/crates/holochain_types/src/app/app_manifest/app_manifest_v1.rs#L165